win32/configure/have_stdckdint.c

```diff
@@ -0,0 +1,42 @@
+/*
+ Check whether stdckdint.h functionality is available.
+ If it's available, then 'have_stdckdint.exe' will be created
+ by config_sh.PL.
+ $Config{i_stdckdint} and the XS symbol I_STDCKDINT will be
+ defined if and only if 'have_stdckdint.exe' prints '0'
+ when executed.
+ Else $Config{i_stdckdint} and I_STDCKDINT will be undef.
+*/
+
+#include <stdio.h>
+#include <stdckdint.h>
+int func_l(long *resultptr, long a, long b) {
```

Comment:
`func_l` is not used outside of this file, so it could be declared `static`.

```suggestion
static int func_l(long *resultptr, long a, long b) {
```